opm/autodiff/AquiferCarterTracy.hpp

```diff
@@ -155,7 +166,7 @@ namespace Opm
             inline void initQuantities(const Aquancon::AquanconOutput& connection)
             {
                 // We reset the cumulative flux at the start of any simulation, so, W_flux = 0
-                W_flux_ = 0.;
```

Comment:
Looks like there may be latent bug already here: do we read/write this on restart? I guess not.

Reply:
> Are you sure about the meaning of the variables? why changing names?

I have no idea about the purpose of this variable, but it accumulates the water flux over time. I changed the name because to me `W_flux_` is not very descriptive of what it does. Suggestions for better names are welcome, though.

> Looks like there may be latent bug already here: do we read/write this on restart? I guess not.

I don't think so, nor do I think we ever did.

Reply:
I changed most (hopefully all) variable names back to their original ones. I am pretty sure most people will now spend more time debugging, though.